.circleci/soltest.sh

```diff
@@ -36,12 +36,44 @@ set -e
 
 OPTIMIZE=${OPTIMIZE:-"0"}
 EVM=${EVM:-"invalid"}
+EOF_VERSION=${EOF_VERSION:-0}
```

Comment:
This change actually made me wonder if we should also consider changing OPTIMIZE and ABI_ENCODER_V1 to integers, even though they are used as boolean values.

Reply:
Would not hurt. The variables should also be quoted. And we should validate the values - currently anything other than `1` means "optimizer disabled", even `"true"`.

I'm going to leave it as is here though. There were just too many small things like this in the script that could be improved and I didn't want to get sidetracked by cleaning it up :)